src/cascadia/TerminalSettingsModel/ApplicationState.h

```diff
@@ -37,6 +37,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 #define MTSM_APPLICATION_STATE_FIELDS(X)                                                                                                                                  \
     X(FileSource::Shared, winrt::hstring, SettingsHash, "settingsHash")                                                                                                   \
     X(FileSource::Shared, std::unordered_set<winrt::guid>, GeneratedProfiles, "generatedProfiles")                                                                        \
+    X(FileSource::Shared, std::unordered_set<winrt::hstring>, GeneratedFolders, "generatedFolders")                                                                       \

```

Comment:
unless we're going to make a habit of this, I think we may just want to have a boolean ;P

we're not gonna make a lot more folders, are we?

Reply:
Figured we'd eventually want to allow extensions to make _folders_ of profiles, but yeah, that's probably a bit too forward thinking. Replaced with a `bool SSHFolderGenerated{ false }` instead